ui/app/components/clients/dashboard.js

```diff
@@ -9,9 +9,28 @@ export default class Dashboard extends Component {
     { key: 'distinct_entities', label: 'unique entities' },
     { key: 'non_entity_tokens', label: 'non-entity tokens' },
   ];
-  @tracked selectedNamespace = null;
```

Comment:
this is now on row 25, alphabetically organized with all the other tracked properties.